src/app/offers/offer-details/offer-details.component.html

```diff
@@ -179,15 +180,20 @@ <h1 class="app-basic-title">{{ states.project.name }}</h1>
                 <div class="investments__container">
                     <div class="investment__total">
                         <div class="total__title">Total Invested</div>
-                        <div class="total__amount">{{ transactionTotal | currencyDefault }}</div>
+                        <div class="total__amount">
+                            <ng-container *ngIf="(investmentTotal$ | async) as investmentTotal else smallSpinner">
+                                {{ investmentTotal | currencyDefault }}
+                            </ng-container>
+                        </div>
                     </div>
-                    <div class="investments__history flex__center-space" *ngFor = "let singleInvestment of investmentData.transactions">
+                    <div class="investments__history flex__center-space"
+                         *ngFor="let singleInvestment of investmentData.transactions">
                         <div class="history__left">
                             <div class="history__amount">
                                 {{ singleInvestment.amount | currencyDefault }}
                             </div>
                             <div class="history__date">
-                                {{ singleInvestment.date | date:'MMMM d y, h:maaa' }}
```

Comment:
consider using builtin date values, as this returned `October 27 2020, 12:1AM` which is not in the correct format (minutes).